lib-es5/v2/uploader.js

```diff
@@ -34,4 +34,5 @@ exports.direct_upload = uploader.direct_upload;
 exports.upload_tag_params = uploader.upload_tag_params;
 exports.upload_url = uploader.upload_url;
 exports.image_upload_tag = uploader.image_upload_tag;
-exports.unsigned_image_upload_tag = uploader.unsigned_image_upload_tag;
\ No newline at end of file
+exports.unsigned_image_upload_tag = uploader.unsigned_image_upload_tag;
+exports.create_slideshow = uploader.create_slideshow;
```

Comment:
No new line at the end of file in this case pollutes diff, line 37 should not have been affected